packages/twenty-server/src/engine/metadata-modules/agent/agent-handoff-executor.service.ts

```diff
@@ -69,14 +68,24 @@ export class AgentHandoffExecutorService {
         );
       }
 
+      const tools = await this.agentToolGeneratorService.generateToolsForAgent(
+        toAgentId,
+        workspaceId,
+      );
+
       const aiRequestConfig = {
         system: targetAgent.prompt,
-        prompt: this.createHandoffPrompt(handoffRequest),
+        messages,
+        tools,
         model: registeredModel.model,
       };
 
       const textResponse = await generateText(aiRequestConfig);
 
+      this.logger.log(
```

Comment:
In v2 let's try to display as much context in the UI as possible to the user, instead of displaying it in the logs